doc/reference/tt_cli/installation.rst

```diff
@@ -0,0 +1,38 @@
+tt CLI installation
+===================
+
+1.  Install third-party software required for building ``tt``:
+
+  * `git <https://git-scm.com/book/en/v2/Getting-Started-Installing-Git>`__,
+    the version control system.
+  * `Go language <https://golang.org/doc/install>`__, version 1.18 or later.
+  * `mage <https://cmake.org/install/>`__ build tool.
```

Comment:
mage != cmake

Reply:
Pasted the wrong link, thank you for spotting this.